index.html

```diff
@@ -1,159 +1,256 @@
-          <li>RSpec</li>
-          <li>Capibara</li>
-          <li>Selenium</li>
-        </ul>
-      </li>
-      <li class="aboutcard">
-        <ul class="grid-item3">
-          <li class="about-title">
-            <h3>Skills</h3>
-          </li>
-          <li>Codekit</li>
-          <li>Codepen</li>
-          <li>Github</li>
-          <li>GitLab</li>
-          <li>terminal</li>
-        </ul>
-      </li>
-    </ul>
-
-  </section>
-</body>
-
-</html>
\ No newline at end of file
+    </section>
+    <section class="about" id="about">
+      <h2 class="aboutheader">About</h2>
+      <p class="introabout">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
+        eiusmod
+      </p>
+      <input type="button" class="btn3" value="Get my CV" />
+      <ul class="grid-container">
+        <li class="aboutcard">
+          <ul class="grid-item1">
+            <li class="about-title">
+              <h3>Languages</h3>
+            </li>
+            <li>JavaScript (es5/es6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item2">
+            <li class="about-title">
+              <h3>Frameworks</h3>
+            </li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item3">
+            <li class="about-title">
+              <h3>Skills</h3>
+            </li>
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>Github</li>
+            <li>GitLab</li>
+            <li>terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <form action="https://formspree.io/f/mayaljan" method="post" id="form">
+      <fieldset class="contact">
+        <h2 class="contactheader">Get Started</h2>
+        <p class="intro">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <label for="name"></label>
+        <input
+          type="text"
+          class="name"
+          id="name"
+          name="name"
+          placeholder="Your Name"
+          required
+        />
+        <br />
+        <label for="name"></label>
+        <input
+          type="text"
+          class="name"
+          id="email"
+          name="email"
+          value="Your Email"
+          required
+        />
+        <br />
+        <label for="message"></label>
+        <textarea
+          name="message"
+          class="message"
+          id="message"
+          placeholder="Enter your message here..."
```

Comment:
- [ ] As the previous reviewer said this text area should be required. kindly let's go ahead and add the `required` attribute to the `<textarea>`.
- [ ]  It would be nice also if you put the padding in your `textarea` so that it looks like the one from the template.